generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>I got the following errors, with a new environment following the guide.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
File /workspace/MONAI/monai/utils/module.py:267, in instantiate(__path, __mode, **kwargs)
    266 if m == CompInitMode.DEFAULT:
--&gt; 267     return component(**kwargs)
    268 if m == CompInitMode.CALLABLE:

File /workspace/MONAI/monai/apps/generation/maisi/networks/diffusion_model_unet_maisi.py:156, in DiffusionModelUNetMaisi.__init__(self, spatial_dims, in_channels, out_channels, num_res_blocks, num_channels, attention_levels, norm_num_groups, norm_eps, resblock_updown, num_head_channels, with_conditioning, transformer_num_layers, cross_attention_dim, num_class_embeds, upcast_attention, use_flash_attention, dropout_cattn, include_top_region_index_input, include_bottom_region_index_input, include_spacing_input)
    155 if use_flash_attention and not has_xformers:
--&gt; 156     raise ValueError("use_flash_attention is True but xformers is not installed.")
    158 if use_flash_attention is True and not torch.cuda.is_available():

ValueError: use_flash_attention is True but xformers is not installed.

The above exception was the direct cause of the following exception:

RuntimeError                              Traceback (most recent call last)
Cell In[10], line 10
      7 checkpoint_autoencoder = load_autoencoder_ckpt(args.trained_autoencoder_path)
      8 autoencoder.load_state_dict(checkpoint_autoencoder)
---&gt; 10 diffusion_unet = define_instance(args, "diffusion_unet_def").to(device)
     11 checkpoint_diffusion_unet = torch.load(args.trained_diffusion_path)
     12 diffusion_unet.load_state_dict(checkpoint_diffusion_unet["unet_state_dict"])

File /workspace/tutorials/generative/maisi/scripts/utils.py:42, in define_instance(args, instance_def_key)
...
    282 warnings.warn(f"Component to instantiate must represent a valid class or function, but got {__path}.")
    283 return component

RuntimeError: Failed to instantiate component 'monai.apps.generation.maisi.networks.diffusion_model_unet_maisi.DiffusionModelUNetMaisi' with keywords: spatial_dims,in_channels,out_channels,num_channels,attention_levels,num_head_channels,num_res_blocks,use_flash_attention,include_top_region_index_input,include_bottom_region_index_input,include_spacing_input
 set '_mode_=debug' to enter the debugging mode.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I can execute the rest of the notebook, after I install xformers locally

Reply:
Please include the following line in readme markdown to instruct users:

```
To run this tutorial, please also install `xformers` by following the [official guide](https://github.com/facebookresearch/xformers#installing-xformers) 
```

Reply:
<p>Can we directly add this in the "Setup environment" cell?</p>

Reply:
Thanks, done